sdk/core/core/README.md

```diff
@@ -50,10 +56,81 @@ typedef struct {
 } az_span;
 ```
 
-## 3. Strings
+### Strings
 
 A string is a span of UTF-8 characters. It's not a zero-terminated string. Defined in [inc/az_str.h](inc/az_str.h).
```

Comment:
This header no longer exists.

Reply:
Fixed, also fixed the structure for az_span